src/javascript/app_2/Utils/Date/date_time.js

```diff
@@ -39,4 +39,29 @@ export const formatDate = (date, date_format = 'YYYY-MM-DD') => moment(date || u
 export const daysFromTodayTo = (date) => {
     const diff = moment(date).utc().diff(moment().utc(), 'days');
     return (!date || diff < 0) ? '' : diff + 1;
-};
\ No newline at end of file
+};
+
+/**
+ * return moment duration between two dates
+ * @param  {Number|moment} epoch start time or moment object
+ * @param  {Number|moment} epoch end time or moment object
```

Comment:
`epoch` or `milliseconds`?

Reply:
if I'm not mistaken [epoch is in milliseconds](https://momentjs.com/docs/#/parsing/unix-timestamp-milliseconds/) :)

Reply:
but yeah we pass epoch as param

Reply:
The term `epoch` refers to [_seconds_](https://momentjs.com/docs/#/parsing/unix-timestamp/) in our API methods. I think the descriptions here will create confusion.

This function expects milliseconds to be passed to (and it seems to be done like that in this PR). However, to create a moment from epoch should be: `moment.unix(Number)`.